badge.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Badge</title>
+</head>
+
+<body>
+    <!-- Avatar Badges -->
+    <div class="badge">
+        <img class="avatar" src="https://source.unsplash.com/3uuLWb6aQXc/" alt="user">
+        <div class="online"></div>
+    </div>
+    <div class="badge">
+        <img class="avatar" src="https://source.unsplash.com/3uuLWb6aQXc/" alt="user">
+        <div class="offline"></div>
+    </div>
+    <div class="badge">
+        <img class="avatar" src="https://source.unsplash.com/3uuLWb6aQXc/" alt="user">
+        <div class="dnd"></div>
+    </div>
+    <!-- Icon Badges -->
+    <div class="badge">
+        <ion-icon class="badge-icon" name="chatbubble"></ion-icon>
+        <span class="number">9</span>
+    </div>
+    <div class="badge">
+        <ion-icon class="badge-icon" name="bag"></ion-icon>
```

Comment:
Suggestion - use google fonts or if you are using any library do mention in your README

Reply:
I'm using google fonts , if you mean font awesome then it also requires cdn import.